Added information about ibexa:install prompting for new password (#2753)
ibexa/documentation-developer: docs/getting_started/install_ibexa_dxp.md

Before executing the command make sure that the database user has sufficient permissions.

The installer will prompt you for a new password for the `admin` user.
Make sure to use a [strong password](security_checklist.md#strong-passwords) meeting all the [password rules](passwords.md#password-rules).

!!! note

> 	In scenarios where entering the new password is not possible, for example in automated deployments and Continuous Integration environments, use the `--no-interaction` option to skip changing the password and keep the default one, `publish`:

Reviewer: ```suggestion
	In scenarios where entering the new password isn't possible, for example, in automated deployments and Continuous Integration environments, use the `--no-interaction` option to skip changing the password and keep the default one, `publish`:
```
